bciers/libs/components/src/datagrid/cells/ChangeUserOperatorStatusColumnCell.tsx

```diff
@@ -21,21 +21,18 @@ const handleUpdateStatus = async (
   statusUpdate: Status,
   roleUpdate: UserOperatorRoles,
 ) => {
-  try {
-    return await actionHandler(
-      `registration/user-operators/${userOperatorId}/update-status`,
-      "PUT",
-      "",
-      {
-        body: JSON.stringify({
-          role: roleUpdate,
-          status: statusUpdate,
-        }),
-      },
-    );
-  } catch (error) {
-    throw error;
-  }
+  const response = await actionHandler(
+    `registration/user-operators/${userOperatorId}/update-status`,
+    "PUT",
+    "",
+    {
+      body: JSON.stringify({
+        role: roleUpdate,
+        status: statusUpdate,
+      }),
+    },
+  );
+  return response;
```

Comment:
We can return the call directly without an await here too

```suggestion
  return actionHandler(
    `registration/user-operators/${userOperatorId}/update-status`,
    "PUT",
    "",
    {
      body: JSON.stringify({
        role: roleUpdate,
        status: statusUpdate,
      }),
    },
  );
```